src/gui/properties/speedplotview.cpp

```diff
@@ -33,15 +33,60 @@
 #include "base/global.h"
 #include "base/utils/misc.h"
 
+namespace
+{
+    enum PeriodInSeconds
+    {
+        MIN1_SEC = 60,
+        MIN5_SEC = 5 * 60,
+        MIN30_SEC = 30 * 60,
+        HOUR6_SEC = 6 * 60 * 60
+    };
+
+    const int MIN5_BUF_SIZE = 5 * 60;
+    const int MIN30_BUF_SIZE = 5 * 60;
+    const int HOUR6_BUF_SIZE = 5 * 60;
+    const int divider30Min = int(MIN30_SEC) / int(MIN30_BUF_SIZE);
```

Comment:
DIVIDER_30MIN
DIVIDER_6HOUR

Reply:
Why do you cast int to int?

Reply:
Tha cast is just leftover grom times when *_BUF_SIZE were enum parts. Removed.

Reply:
It was meaningless even at that time, because enum declares constants of type int.

Reply:
Enum divide on enum gave me compiler warning. Or error. Anyway, I can not reproduce it in new place, so removed cast from enum too.